### PR TITLE
Remove let and var macros now that we're all-C++

### PR DESCRIPTION
**Pull request by Adlai-Holler**
Not only are these a little icky but the label "var" is used in Abseil header functions which causes it to choke when imported after Texture.

### CODE REVIEW

File: Source/ASCollectionView.mm [hunk 1]
```diff
@@ -1551,7 +1551,7 @@ - (BOOL)collectionView:(UICollectionView *)collectionView canMoveItemAtIndexPath
     return NO;
   }
 
-  // If the data source implements canMoveItem, const auto them decide.
+  // If the data source implements canMoveItem, let them decide.
```
Comment (nguyenhuy) [inline Source/ASCollectionView.mm]: “constabt define them decide” makes a good joke though lol

**Comment by Adlai-Holler** [comment]
Could someone const auto @garrettmoon know the CI is in trouble?

**Comment by nguyenhuy** [comment]
Done. Might take until early next week for him/Ricky to kick the machine though.

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/9366823828e76afa2bb4c60ae32aafa0/raw/d40e0fd67e8d12ba0187e2dd8e9246e92351033a/log.txt)

**Comment by pinterest-ios** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: GET https://api.github.com/repos/TextureGroup/Texture/contents/Tests/ASTLayoutFixture.mm?ref=227a5a1e9496130737d419447ec47fd0c580dc34: 502 - Server Error. Updating the Danger gem might fix the issue. Your Danger version: 5.5.10, latest Danger version: 5.11.1

```
 #  from Dangerfile:43
 #  -------------------------------------------
 #        data = ""
 >        contents = github.api.get file["contents_url"]
 #        open(contents["download_url"]) { |io|
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/b53bcea4180787684a4df8c590338372/raw/4663bf34062ba74f9b009df2ed17526c48e960fd/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/26c3d3671a8f1fd517e058803cbfedb2/raw/ab466ef8148df348c7065604274a1dc3733150f0/log.txt)

**Pull request merged**